Exiled.Events/Patches/Events/Player/FlippingCoin.cs

```diff
@@ -39,19 +39,7 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
             LocalBuilder ev = generator.DeclareLocal(typeof(FlippingCoinEventArgs));
 
             int offset = 0;
-            int index = newInstructions.FindIndex(instruction => instruction.opcode == OpCodes.Ldc_I4_S) + offset;
-
-            // Extract the old labels, before removing it
-            List<Label> oldLabels = newInstructions[index].ExtractLabels();
-
-            // Remove both NW events and logic
-            newInstructions.RemoveRange(index, 45);
-
-            // Redirect the old labels to the new starting point
-            newInstructions[index].WithLabels(oldLabels);
-
-            offset = -1;
-            index = newInstructions.FindLastIndex(instruction => instruction.LoadsField(Field(typeof(Coin), nameof(Coin.RateLimiter)))) + offset;
+            int index = newInstructions.FindIndex(instruction => instruction.LoadsConstant(109)) + offset;
```

Comment:
If the flip gets prevented from the NwAPI `PlayerPreCoinFlip` event, the Exiled event is never called.
We could either suppress their event (not ideal) or remove the return from their switch and add their `CoinFlipCancellation` enum to our event, to handle its logic by ourselves.

Reply:
Where is it?